.github/actions/deploy-lifecycle-manager-e2e/action.yml

```diff
@@ -10,12 +10,20 @@ inputs:
 runs:
   using: composite
   steps:
+    - name: Set Kustomize Directory
+      shell: bash
+      run: |
+        if [[ "${E2E_USE_GARDENER_CERT_MANAGER}" == "1" ]]; then
```

Comment:
I don't think we need to be that explicit and check against `"1"` 😄 
`if [[ -n "${E2E_USE_GARDENER_CERT_MANAGER}" ]]; then` should be enough for the control flow here (set vs unset)

Same for other occurrences in this PR.